indico/modules/events/abstracts/templates/abstract/timeline.html

```diff
@@ -74,33 +76,79 @@
 {% macro _render_review(review, form, management=false) %}
     <div id="abstract-review-{{ review.id }}" class="i-timeline-item">
         <div class="i-timeline-item-label action abstract-comment-pic"></div>
-        <div class="i-timeline-item-box header-indicator">
-            <div class="i-box-header flex">
-                <div class="stretch">
-                    {% trans name=review.user.name -%}
-                        <strong>{{ name }}</strong> left a review
-                    {%- endtrans %}
-                    {{ render_visibility_hint(review.visibility) }}
-                    <time datetime="{{ review.created_dt.isoformat() }}">
-                        {{- review.created_dt|format_human_date -}}
-                    </time>
-                    {% if review.modified_dt %}
-                        {{ render_edited_hint(review.modified_dt) }}
-                    {% endif %}
+        <div class="flex i-timeline-item-content">
+            <div class="i-timeline-item-metadata">
+                <div class="flex">
+                    <span class="abstract-metadata-action">
+                        {% trans name=review.user.name -%}
+                            <strong>{{ name }}</strong> left a review
+                        {%- endtrans %}
+                        {{ render_visibility_hint(review.visibility) }}
+                        <time datetime="{{ review.created_dt.isoformat() }}">
+                            {{- review.created_dt|format_human_date -}}
+                        </time>
+                        {% if review.modified_dt %}
+                            {{ render_edited_hint(review.modified_dt) }}
+                        {% endif %}
+                    </span>
+                    <span class="abstract-metadata-context">
+                        {{ render_track(review.track, classes='abstract-track-tag i-tag outline') }}
+                    </span>
                 </div>
-                <div class="abstract-comment-badges flex">
```

Comment:
This breaks hiding the sentence, score and action icons when editing the review.